Assets/Scripts/ScriptableObjects/TestingControllers/MouseLook.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class MouseLook : MonoBehaviour
```

Comment:
This is definitely something that should be built using Cinemachine


https://docs.unity3d.com/Packages/com.unity.cinemachine@2.1/api/Cinemachine.CinemachinePOV.html

Reply:
I don't know what Cinemachine is will look into it, was just following along with a tutorial, I did not end up using this script, it was one the testing ones.